bin/database.py

```diff
@@ -1164,6 +1164,12 @@ def upgrade_to_34():
     config.db.groups.update_many({'roles': {'$exists': True}}, {'$rename': {'roles': 'permissions'}})
     config.db.groups.update_many({'name': {'$exists': True}}, {'$rename': {'name': 'label'}})
 
+def upgrade_to_35():
+    """
+    Indexes the timestamps of access logs
+    """
+    config.log_db.access_log.createIndex({'timestamp': -1})
```

Comment:
You can put this in config.py and it will get added on startup if it's not already there.